backend/coins/eth/etherscan/etherscan.go

```diff
@@ -67,9 +68,13 @@ func (etherScan *EtherScan) call(params url.Values, result interface{}) error {
 	if response.StatusCode != http.StatusOK {
 		return errp.Newf("expected 200 OK, got %d", response.StatusCode)
 	}
-	if err := json.NewDecoder(response.Body).Decode(result); err != nil {
+	body, err := ioutil.ReadAll(response.Body)
```

Comment:
I would limit here with `io.LimitedReader`. Not sure but 1Mb is probably more than enough?

Reply:
maybe another day - don't want to spend time now estimating the max size